device/api/umd/device/cluster.h

```diff
@@ -669,7 +669,7 @@ class Cluster : public tt_device {
     Cluster(
         const std::string& sdesc_path,
         const std::set<chip_id_t>& target_devices,
-        const uint32_t& num_host_mem_ch_per_mmio_device = 1,
```

Comment:
nit: are you removing reference here just because it is a basic type?

Reply:
Yes -- there's no need for this to be passed by reference.